### ISSUE TITLE
Google_distance() Time outs

### ISSUE THREAD

**Issue by nailez** [SymbolixAU/googleway#20]
My team are currently building a tool that uses your googleway R code.
We LOVE IT! so thanks for putting it together.
We've hit a bit of a snag however. 
There seems to be some issue with timeouts.
We are attempting to get drive time on 140k records for our exercise. 
We've successfully got results on approx 20k records but it has timed out twice.
The first time we didn't check the run time, but the 2nd time it ran for ~2 hours and then timed out.
I wonder if you know if there's a setting we can tweak to avoid the time out so that our job runs to completion and could help us doctor the code a little to allow this to occur?

**Comment by SymbolixAU** [comment]
A potential solution for the package is to use Google's 'Timeout and Errors' - https://developers.google.com/api-client-library/java/google-api-java-client/errors

Alternatively a `httr::status_code()` check. Or a try catch statement in the R code around the `google_xxx()` funciton. 

Currently this is a low priority for me to add into the package. But you're welcome to issue a Pull Request if you come up with a solution.

**Comment by SymbolixAU** [comment]
@nailez have you had any success implementing a solution to this issue?

**Comment by nailez** [comment]
@SymbolixAU Nope not yet. We had to move onto other things. Someone on the community page tells me that i'm not on a premium account. Which may be true but runs counter to the fact that I had thought I'd hit the "go premium" button when prompted. In summary though, I'm no closer to a solution at this point. My team will definitely get back to sorting it out as it's a very useful tool for us but it may not be for a month or two.
I did use the tryCatch function to manage to get the majority of records tagged so thanks for that tip.
